index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="assets/css/navbar-crafted.css" />
   </head>
   <body>
+    <!-- Start Navbar Crafted -->
```

Comment:
I think it would be cleaner to leave the comments out. The html is already sectioned so it seems clear enough from the code itself